setup.py

```diff
@@ -20,7 +20,7 @@
     install_requires=[
         'pandas',
         'windows-curses ; platform_system=="Windows"',
-	'pandasql'
```

Comment:
I'm curious why we have a list of dependencies in requirements.txt and then only a subset here? Can one of you explain @johncmerfeld @baogorek

Reply:
My understanding is that setup.py is a spec doc for packages, whereas requirements.txt is a tool to exactly recreate an environment at a given time. If you run `pip install .`, then you get the most recent versions of all the packages required by setup.py plus perhaps some dependencies. I actually think we should install with `pip install .` in the CircleCI build since that will simulate how people will be installing it in the wild. On the question of whether there should be any dependency left out of setup.py (say something that gets automatically installed by Pandas), I don't know for sure. An engineer at Google told me it's best practice to declare even implicit dependencies, but I feel I've seen pretty lightweight setup.py files in the wild. (BTW, you can see that I added pytest and flake8 in setup.py in the circle-ci branch under "install_extras.")